app/forms/candidate_interface/gcse_not_completed_form.rb

```diff
@@ -9,7 +9,7 @@ class GcseNotCompletedForm
 
     validates :not_completed_explanation, presence: true, if: -> { currently_completing_qualification }
 
-    validates :not_completed_explanation, word_count: { maximum: 200 }
+    validates :not_completed_explanation, length: { maximum: 256 }
```

Comment:
Do the views need to be changed?

https://github.com/DFE-Digital/apply-for-teacher-training/blob/366f376d0238017821eb628b708ae75c943b9f97/app/views/candidate_interface/gcse/not_completed_qualification/_form.html.erb#L8

Reply:
Oh no, I must have missed that one! Good catch.

Reply:
You did do this. My fault, sorry 🤦🏼‍♀️